libstratum/EthStratumClient.cpp

```diff
@@ -502,6 +502,16 @@ void EthStratumClient::work_timeout_handler(const boost::system::error_code& ec)
 	}
 }
 
+bool EthStratumClient::submitHashrate(string const & rate) {
+	h256 id = h256::random();
```

Comment:
Based on the documentation, its a random id needed as param2 for submitting hashes.
The farm Client uses this as second parameter, initialized in ``MinerAux.h``.
I can do it the same here, but since the id is used only for an out of spec method i thought it make sense to not init it out of class. But i'm happy to move it to a private class var, initialised in the constructor.